apps/service/package.json

```diff
@@ -26,7 +25,7 @@
     "json-rpc-2.0": "^1.7.0",
     "lodash": "^4.17.21",
     "reflect-metadata": "^0.2.0",
-    "rgbpp": "0.0.0-snap-20240813134030",
+    "rgbpp": "workspace:*",
```

Comment:
Since we're referencing the `workspace:*` version of the `rgbpp`, it's required to run the command `pnpm run build:packages` before executing the `start:dev` or `build` commands. Should we mention this in the README? Does anything in the deployment files (e.g. docker-compose) need to be updated?

Reply:
Agree.  Maybe we can release v0.6.0 this week and update the dependency.